Dockerfile

```diff
@@ -1,37 +1,39 @@
-FROM phusion/baseimage:0.9.19
+FROM phusion/baseimage:0.9.22
 
-# Use baseimage-docker's init system.
-CMD ["/sbin/my_init"]
+# Make available on port 8000 - however, it is not public yet
+# To make it public, use -p flag
+EXPOSE 8000
 
 # Directory containing ADAGE server source code
 ENV ADAGE_SRC=adage
+
 # Directory used in container
 ENV ADAGE_SRV=/srv
-# Directory where ADAGE code lives in the container
-ENV ADAGE_SRVSRC=$ADAGE_SRV/$ADAGE_SRC
+
+# Create required directories
+WORKDIR $ADAGE_SRV
```

Comment:
I agree with this, I think I echoed a similar thought when Rene and I originally paired on this Dockerfile.